README.md

```diff
@@ -284,6 +285,11 @@ A collection of awesome things regarding React ecosystem.
 * [Reactotron: Control, monitor, and instrument](https://github.com/skellock/reactotron)
 * [React-fix-it: Automagically generate tests from errors](https://github.com/MicheleBertoli/react-fix-it)
 
+##### Analyzing React Performance
+* [Introducing the React Profiler](https://reactjs.org/blog/2018/09/10/introducing-the-react-profiler.html)
```

Comment:
Could @tsiq-swyx‘ netlify blog post be added? I thought it was nice too :)

Reply:
Hey @Haroenv Did you mean this one?
https://www.netlify.com/blog/2018/08/29/using-the-react-devtools-profiler-to-diagnose-react-app-performance-issues/

Reply:
yep!